examples/cassandra/README.md

```diff
@@ -32,7 +33,19 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 
-## Cloud Native Deployments of Cassandra using Kubernetes
+# Cloud Native Deployments of Cassandra using Kubernetes
+
+## Table of Contents
+
+  - [Prerequisites](#prerequisites)
+  - [tl;dr Quickstart](#tldr-quickstart)
+  - [Step 1: Create a Cassandra Service](#step-1-create-a-cassandra-service)
+  - [Step 2: Use a Replication Controller to create Cassandra node pods](#step-2-use-a-replication-controller-to-create-cassandra-node-pods)
+  - [Step 3: Scale up the Cassandra cluster](#step-3-scale-up-the-cassandra-cluster)
+  - [Step 4: Delete the Replication Controller](#step-4-delete-the-replication-controller)
+  - [Step 5: Use a DaemonSet instead of a Replication Controller](#step-5-use-a-daemonset-instead-of-a-replication-controller)
+  - [Step 6: Resource Cleanup](#step-6-resource-cleanup)
+  - [Seed Provider Source](#seed-provider-source)
 
 The following document describes the development of a _cloud native_
```

Comment:
This paragraph reads a bit awkward and is wordy.  Since it is the first paragraph in the document we should work to make sure it is clean and clear.  I would also prefer if as much as possible, our examples have the same wording and structure to minimize the amount of mental effort required when switch between examples.

See https://github.com/jeffmendoza/kubernetes/blob/wordpress-update/examples/mysql-wordpress-pd/README.md

```
This example describes how to run a persistent installation of WordPress and MySQL on Kubernetes. We'll use the mysql and wordpress official Docker images for this installation. (The WordPress image includes an Apache server).

Demonstrated Kubernetes Concepts:

Persistent Volumes to define persistent disks (disk lifecycle not tied to the Pods).
Services to enable Pods to locate one another.
External Load Balancers to expose Services externally.
Deployments to ensure Pods stay up and running.
Secrets to store sensitive passwords.
```

If this were to match the wordpress example more closely, it might read something like

```
This example describes how to run a cloud native Cassandra deployment on Kubernetes.  We'll use a custom Cassandra image built with a Kubernetes-native Cassandra `SeedProvider`.

Demonstrated  Kubernetes Concepts:
Services for X
...
```

Also the layout of the various sections differs from wordpress example.  If we think one structuring is better than the other, I am fine with changing the canonical format, but would like to see the structures more standardized.  Once we settle on the exact details, we should probably update the guidelines with a template that everything should match.